File: plugin/bukkit/src/main/java/com/jonathanpotts/cartographytable/bukkit/CartographyTable.java
package com.jonathanpotts.cartographytable.bukkit;

import org.bukkit.plugin.java.JavaPlugin;

/**
 * The CartographyTable server plugin.
 */
public class CartographyTable extends JavaPlugin {
  @Override
  public void onEnable() {
    getCommand("refresh-map-data").setExecutor(new CommandRefreshMapData(this));
  }
}

File: src/main/kotlin/io/github/jonathanpotts/cartographytable/CartographyTable.kt
package io.github.jonathanpotts.cartographytable

import org.bukkit.plugin.java.JavaPlugin

/**
 * The CartographyTable server plugin.
 */
@Suppress("unused")
class CartographyTable : JavaPlugin() {
    override fun onEnable() {
        getCommand("refresh-map-data")?.setExecutor(CommandRefreshMapData(this))
    }
}
